repos/system_upgrade/el7toel8/actors/vimcheck/actor.py

```diff
@@ -0,0 +1,82 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import InstalledRedHatSignedRPM, VimModel, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+vimconfigs = {
+              'vim-minimal': '/etc/virc',
+              'vim-enhanced' : '/etc/vimrc'
+              }
+
+changes = [
+            'let skip_defaults_vim=1',
+            'set t_BE='
+          ]
+
+
+
+class VimCheck(Actor):
+    """
+    Actor for checking if vim is installed and if vimrc file exists.
+    """
+
+    name = 'vim_check'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report, VimModel)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def checkconfigs(self, packages_dict):
+      """
+      checkconfigs
+
+      Vim project has two configs - /etc/virc for vi and /etc/vimrc for vim.
+      This method appends respective configuration filename into list according to
+      installed Vim packages.
+
+      parameters:
+      - packages_dict - dictionary
+
+      returns list of strings
+      """
+      configs = []
+
+      for package in packages_dict:
+        if packages_dict[package] == True:
```

Comment:
Do not check `True` in such way. Use `is` and `not is` for such cases.

```suggestion
        if packages_dict[package] is True:
```